lib/widgets/info_card.dart

```diff
@@ -18,85 +19,49 @@ class InfoCard<T> extends StatelessWidget {
   final String title;
   final ValueKey<String> showMoreButtonKey;
   final EdgeInsetsGeometry padding;
+  final bool showIfEmpty;
```

Comment:
```suggestion
  // If true, the card is displayed with a placeholder text ("None yet."). Otherwise, the card is hidden.
  final bool showIfEmpty;
```